test/src/main/java/io/strimzi/test/k8s/Oc.java

```diff
@@ -42,7 +46,7 @@ public void close() {
 
     @Override
     public Oc clientWithAdmin() {
```

Comment:
We are using `client.clientWithAdmin()` few times. It means that we are performing actions as `system:admin`. Remotely I am not able to achieve this (guess it's not possible?) I tried remove all this calls but there is admin context creation in `BaseKubeClient.java` at line 216 (context is not used anyway). Maybe we can remove it and set admin rights to developer user ?

```
io.strimzi.test.k8s.KubeClusterException: `oc login -u system:admin` got status code 1 and stderr:
------
error: username system:admin is invalid for basic auth```

Reply:
I haven't tried using OpenShift.  However, I would recommend setting up a context with sufficient rights to create and delete namespaces on the cluster in use.  I don't think this additional code accomplishes anything that granting rights in this context won't do.

Reply:
User has correct rights in cluster. Problem is, that our test suite is trying to use `system:admin` user which can be used only as a root user on machine where cluster is running or with `kubectl` config file from machine where cluster is. Namespace creation is working because it's overridden in `Oc.java` and use user `developer` for it, but deletion is implemented only in `BaseKubeClient.java` so client try to use user `system:admin` which is not correct for remote actions. I recommend to remove `system:admin` user from `adminContext` [here](https://github.com/strimzi/strimzi-kafka-operator/blob/55a37780be20c08ab9fde023e94b4c303eb124c5/test/src/main/java/io/strimzi/test/k8s/Oc.java#L35).  What do you think @tombentley  ?